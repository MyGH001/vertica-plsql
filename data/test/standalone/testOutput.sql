/**
* testing output message with print and dbmd_output 
*/

create or replace procedure p_test_output()
as
begin
  print('Built for Fast.');
  for r IN (select 'Built' as word union all select 'for' union all select 'Freedom.') loop
    dbms_output.put_line(r.word);
  end loop;
end;

begin
  call p_test_output();
end;
