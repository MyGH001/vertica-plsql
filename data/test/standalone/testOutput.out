Built for Fast.
Built
for
Freedom.
